### ISSUE TITLE
Redirection after oauth SIGNUP

### ISSUE THREAD

**Issue by Uzay-G** [publiclab/plots2#7211]
After #7113, oauth should be correctly redirecting to the previous page the user was on before he logged in/signed up.

Now there has been lots of discussion around where users should be redirected after signing up. This was summed up by @jywarren in #6233:

> I'm thinking about the discussion of whether on SIGNUP we want to leave people on the dashboard or on the page they came from, so that's unrelated to this issue on LOGIN.
> 
> Signup-related issues have sometimes been opened to suggest that people end up back where they came from but my understanding is that they need to arrive on the dashboard to see the "welcome message". Related issues:
> 
>     #3367
>     #5600 (not sure if this is signup or login)
> 
> But, possibly related to this issue: #3246
> 
> Thanks!

For now, signing up through oauth will redirect to the previous page the user was on. But I think we should discuss this issue of redirection for signing up.

I would be happy to change the code to redirect the user to the dashboard after signing up through oauth.

What do you think @jywarren and @SidharthBansal?

**Comment by Uzay-G** [comment]
@SidharthBansal can you add this to the login/signup milestones? Thanks :+1:

**Comment by SidharthBansal** [comment]
do we need to solve this before merging other prs?
Or can we do this later?
Any idea?

**Comment by Uzay-G** [comment]
We can solve this after #7113. We need to discuss how we want this redirection to work and then implement the changes.

**Comment by SidharthBansal** [comment]
@Uzay-G can we proceed with this now?
Which tests were you going to add?
Please open a new issue so that we can discuss them there.

**Comment by Uzay-G** [comment]
Yes, if we want to redirect to the dashboard after oauth signup we should work on this. Otherwise we can leave it as is and the user will be redirected to the previous page when signing up. I will open a new issue for the tests.

**Comment by SidharthBansal** [comment]
Can you please close this issue and open up a new issue after refactoring #7211?

**Comment by SidharthBansal** [comment]
We just need to migrate as it is difficult to understand things here

**Comment by Uzay-G** [comment]
You want me to close this one and open a new one after refactoring which pr/issue?

**Comment by SidharthBansal** [comment]
7211

**Comment by Uzay-G** [comment]
Sorry I don't understand this issue is 7211

**Comment by SidharthBansal** [comment]
Just I need you to reformat the current issue for view from Jefd

**Comment by SidharthBansal** [comment]
Moved to #7240

**Comment by Uzay-G** [comment]
Sorry I was going to work on this but didn't have any time today. Thank you so much for opening the issue!
